packages/hui-pages/src/containers/GlobalStylesInjector.js

```diff
@@ -82,6 +82,10 @@ body {
 *:after {
   box-sizing: inherit;
 }
+
+a {
+  text-decoration: none;
+}
```

Comment:
Is this to override the `<Link />` styles?

Reply:
It's to remove underlines that we've never had on links.  It's the default style for links, not <Link /> specifically

Reply:
Cool